projects/lib/src/buyer/components/page-renderer/page-renderer.component.ts

```diff
@@ -159,14 +159,34 @@ export class PageRendererComponent implements OnChanges, AfterViewInit {
     const target = component.document.getElementsByTagName(appendTo)[0];
 
     if (content) {
-      const element = $(content);
-      const scripts = element.filter(function() {
+      let element;
+      try {
+        element = $(content);
+      } catch {
```

Comment:
I'd recommend logging the error instead of assuming its always due to special characters. That may be the case most of the time but may be misleading and hard to track down if its ever something else. 

You can keep your other logging if you like but maybe just log the raw error in addition to it.

Reply:
good call, i like that idea. Just pushed up some changes for that.

I ended up deleting the log about the plain text because after doing more debugging, I realized that plain text (without special characters) didn't break anything. This is because `element.length` will be 0, meaning `scripts` and `nonScripts` values are going to have length of 0 as well. So then nothing will get appended to the header/footer embeds in this case.